2_adventures/001/jimmylorunning/01_day1/README.md

```diff
@@ -0,0 +1,13 @@
+- `git checkout master`
+- `git remote add upstream https://github.com/RubySteps/21-day-challenge`
+- `git remote -v` (to make sure it's there)
+- `git pull upstream master` (to get new changes since I last pulled)
+- `git checkout -b 01_day1`
+- `mkdir -p 2_adventures/001/jimmylorunning/01_day1/'`
+- `vim !$README.md`
```

Comment:
Very nice use of the !$ last argument variable in bash!  (Just an explanation for those that may not be aware)